analyzer/Makefile

```diff
@@ -22,4 +26,97 @@ venv_dev:
 clean_venv_dev:
 	rm -rf venv_dev
 
-# TODO: add targets to build analyzer separately.
+package_dir_structure:
+	mkdir -p $(BUILD_DIR) && \
+	mkdir -p $(CC_BUILD_DIR)/bin && \
+	mkdir -p $(CC_BUILD_LIB_DIR)
+
+build_plist_to_html:
+	$(MAKE) -C $(ROOT)/tools/plist_to_html
+
+package_plist_to_html: build_plist_to_html package_dir_structure
+	# Copy plist-to-html files.
+	cp -r $(ROOT)/tools/plist_to_html/plist_to_html $(CC_BUILD_LIB_DIR) && \
+	cp -r $(ROOT)/tools/plist_to_html/build $(CC_BUILD_DIR)/plist_to_html && \
+	ln -sf $(CC_BUILD_DIR)/plist_to_html/bin/plist-to-html $(CC_BUILD_DIR)/bin/plist-to-html
+
+build_tu_collector:
+	$(MAKE) -C $(ROOT)/tools/tu_collector
+
+package_tu_collector: build_tu_collector package_dir_structure
+	# Copy tu_collector files.
+	cp -r $(ROOT)/tools/tu_collector/tu_collector $(CC_BUILD_LIB_DIR) && \
+	cp -r $(ROOT)/tools/tu_collector/build $(CC_BUILD_DIR)/tu_collector && \
+	ln -sf $(CC_BUILD_DIR)/tu_collector/bin/tu-collector $(CC_BUILD_DIR)/bin/tu-collector
+
+package_analyzer: package_dir_structure
```

Comment:
Please add a comment that this target should be used from the top level `Makefile` to build the package together with the web part.